docs/src/reference/glossary.rst

```diff
@@ -2,51 +2,55 @@
   :description: Tensile is a tool for creating a benchmark-driven backend library for GEMM
   :keywords: Tensile kernel selection, Tensile solution selection, GEMM, Tensor, ROCm
 
-.. _nomenclature:
+.. _glossary:
 
-************
-Nomenclature
-************
+*********
+Glossary
+*********
+
+This topic lists and describes the frequently used terms in the Tensile documentation.
```

Comment:
This isn't actually a glossary. To me, a glossary is an alphabetical list of terms, which this isn't.  I think this should be reverted to "Nomenclature" as it's a document that helps clarify the system of naming used within the project.

There are some terms that could be added to a glossary section at the end of this file, that's fine with me.